### PR TITLE
Updated for v2.8.0

### PR DESCRIPTION
**Pull request by francislavoie**
Still WIP

### CODE REVIEW

File: src/docs/markdown/caddyfile/directives/handle_errors.md [hunk 1]
```diff
@@ -18,11 +20,13 @@ You may use the [`error`](error) directive to explicitly trigger an error based
 ## Syntax
 
 ```caddy-d
-handle_errors {
+handle_errors <status_codes...> {
```
Comment (mohammed90) [inline src/docs/markdown/caddyfile/directives/handle_errors.md]: It's optional, right?

```suggestion
handle_errors [<status_codes...>] {
```

File: src/docs/markdown/caddyfile/directives/file_server.md [hunk 2]
```diff
@@ -33,7 +33,9 @@ file_server [<matcher>] [browse] {
 	root          <path>
```
Comment (mohammed90) [inline src/docs/markdown/caddyfile/directives/file_server.md:33]: Should we somehow mention the `embedded` filesystem?

Comment (francislavoie) [reply src/docs/markdown/caddyfile/directives/file_server.md:33]: Probably yeah. Do you want to write a blurb about it?

File: src/docs/markdown/caddyfile/directives/handle_errors.md (hunk 1 above)
Comment (francislavoie) [reply src/docs/markdown/caddyfile/directives/handle_errors.md]: Good call

File: src/docs/markdown/caddyfile/directives/file_server.md (hunk 2 above)
Comment (mohammed90) [reply src/docs/markdown/caddyfile/directives/file_server.md:33]: Will do. I'll include it when I visit this branch for my parts later.

File: src/docs/markdown/caddyfile/directives/rewrite.md [hunk 3]
```diff
@@ -23,6 +23,8 @@ rewrite [<matcher>] <to>
 
 - **&lt;to&gt;** is the URI to rewrite the request to. Only the components of the URI (path or query string) that are specified in the rewrite will be operated on. The URI path is any substring that comes before `?`. If `?` is omitted, then the whole token is considered to be the path.
 
+Prior to v2.8.0, that the `<to>` argument could be confused by the parser for a [matcher token](/docs/caddyfile/matchers#syntax) if it began with `/`, so it was necessary to specify a wildcard matcher token (`*`).
```
Comment (mohammed90) [inline src/docs/markdown/caddyfile/directives/rewrite.md]:
```suggestion
Prior to v2.8.0, the `<to>` argument could be confused by the parser for a [matcher token](/docs/caddyfile/matchers#syntax) if it began with `/`, so it was necessary to specify a wildcard matcher token (`*`).
```

File: src/docs/markdown/caddyfile/directives/root.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ root [<matcher>] <path>
 
 - **&lt;path&gt;** is the path to use for the site root.
 
-Note that the `<path>` argument could be confused by the parser for a [matcher token](/docs/caddyfile/matchers#syntax) if it begins with `/`. To disambiguate, specify a wildcard matcher token (`*`). See examples below.
+Prior to v2.8.0, that the `<path>` argument could be confused by the parser for a [matcher token](/docs/caddyfile/matchers#syntax) if it began with `/`, so it was necessary to specify a wildcard matcher token (`*`).
```
Comment (mohammed90) [inline src/docs/markdown/caddyfile/directives/root.md]:
```suggestion
Prior to v2.8.0, the `<path>` argument could be confused by the parser for a [matcher token](/docs/caddyfile/matchers#syntax) if it began with `/`, so it was necessary to specify a wildcard matcher token (`*`).
```

**Comment by francislavoie** [comment]
I'll merge this as-is for now, but we're not done yet with all the 2.8 docs. Good start.

**Comment by mholt** [comment]
This is awesome already, thanks!! _GREAT_ start.

**Pull request merged**